controller/gui/template/network.mustache

```diff
@@ -34,20 +39,27 @@
                 </span>
             </summary>
 
-            <pre class="d-Preformatted">{{properties}}</pre>
+            <div class="d-Network__Properties">
+                <pre class="d-Preformatted">{{properties}}</pre>
+
+                {{^connected}}
+                    <form action="/network/{{id}}/connect" method="post" class="d-Network__Form">
+                        <label for="passphrase" class="d-Network__Label">Passphrase</label>
+                        <input type="text" class="d-Input d-Network__Input" id="passphrase" name="passphrase">
+
+                        <label for="proxy" class="d-Network__Label">Proxy</label>
+                        <input type="text" class="d-Input d-Network__Input" id="proxy" value="{{proxy}}" name="proxy" placeholder="192.168.1.10:8500">
 
-            {{^connected}}
-                <form action="/network/{{id}}/connect" method="post" class="d-Network__Form">
-                    <input type="text" class="d-Input d-Network__Input" name="passphrase">
-                    <input type="submit" class="d-Button" value="Connect">
-                </form>
-            {{/connected}}
+                        <input type="submit" class="d-Button d-Network__Button" value="Connect">
+                    </form>
+                {{/connected}}
```

Comment:
Would it be hard to still allow for the proxy to be set on an already configured network? It's a bit awkward to have to remove the network in order to configure a proxy (and then have to retype the password).

Reply:
It’s a matter of adding back the previous method: https://github.com/dividat/playos/pull/44/commits/ac6e4f1788c69cac311ac1c3f0fc5b006536cdfb.